test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -227,10 +196,12 @@
 
 (deftest ^:integration hold-file-handle-on-instance-while-pool-flush-in-progress-test
   (testing "file handle opened from old pool instance is held open across pool flush"
-    (bootstrap/with-puppetserver-running
+    (tk-testutils/with-app-with-config
       app
-      {:jruby-puppet {:max-active-instances 2
-                      :borrow-timeout default-borrow-timeout}}
+      [jruby/jruby-puppet-pooled-service]
+      (jruby-testutils/jruby-puppet-tk-config
+       (jruby-testutils/jruby-puppet-config {:max-active-instances      4
```

Comment:
Dunno, but this whole test suite only takes like 30 seconds to run, so I'm not sweating it ATM.